ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/_cq_dialog/.content.xml

```diff
@@ -267,32 +267,33 @@
                                 <items jcr:primaryType="nt:unstructured">
                                     <validationPatternType
                                             jcr:primaryType="nt:unstructured"
+                                            granite:class="cmp-adaptiveform-textinput__validationpattern"
                                             sling:resourceType="granite/ui/components/coral/foundation/form/select"
                                             fieldLabel="Type"
-                                            name="./validationPatternType">
+                                            name="./validatePatternType">
+                                        <datasource
+                                                jcr:primaryType="nt:unstructured"
+                                                sling:resourceType="core/fd/components/form/container/v1/datasource"
+                                                fieldType="text-input"
+                                                type="formatters"/>
                                     </validationPatternType>
-                                    <validatePictureClause
+                                    <validateFormat
                                             jcr:primaryType="nt:unstructured"
+                                            granite:class="cmp-adaptiveform-textinput__validationformat"
                                             sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
                                             fieldDescription="Regular Expression is permitted."
                                             fieldLabel="Pattern"
-                                            name="./pattern">
-                                    </validatePictureClause>
+                                            name="./validateFormat">
+                                    </validateFormat>
                                     <validatePictureClauseMessage
                                             jcr:primaryType="nt:unstructured"
                                             sling:resourceType="granite/ui/components/coral/foundation/form/textarea"
                                             fieldLabel="Error Message"
-                                            name="./validatePictureClauseMessage"/>
+                                            name="./validatePatternMessage"/>
```

Comment:
sling models still read validatePictureClauseMessage  
See: https://github.com/adobe/aem-core-forms-components/blob/master/bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java#L332

Reply:
reverted this